src/config.ts

```diff
@@ -41,33 +41,29 @@ export const config = yargs
   .option('quote-in-mock', {
     description:
       'Mock data to use for a transfer in quote that should be offered',
-    type: 'string',
     demandOption: true,
     example: 'quoteInNigeriaCUSD',
-    options: Object.keys(MOCK_QUOTE),
+    choices: Object.keys(MOCK_QUOTE),
```

Comment:
options are optional, choices are mandatory. this short circuits if an invalid config param is given, which leads to clearer test failures

Reply:
by "options are optional" you mean that this strictly doesn't require to match the fields in options?